keyboards/cradio/config.h

```diff
@@ -70,7 +59,13 @@
 /* Top left key on left half */
 #define BOOTMAGIC_LITE_ROW    0
 #define BOOTMAGIC_LITE_COLUMN 0
-/* Top right key on right half */
-#define BOOTMAGIC_LITE_ROW_RIGHT    4
-#define BOOTMAGIC_LITE_COLUMN_RIGHT 4
 
+/* RP2040 defaults */
+#ifdef MCU_RP
+#    define SPLIT_USB_DETECT
+#    define SERIAL_PIO_USE_PIO1
```

Comment:
Is PIO0 used by a different PIO program and therefore not available?

Reply:
PIO0 worked fine. Simplified both files, including @drashna's suggestion.